### DISCUSSION TITLE
iCUE and Corsair Service question.

### DISCUSSION THREAD

**Discussion by Grid21** [EvanMulawski/FanControl.CorsairLink#116] [General]
I read in the plugin description that iCUE and Corsair Service should not be running while Fan Control is using it. However, my question is, if neither is running, then I assume that means the LED control for my mouse, keyboard, and ram, will stop functioning as well? Is that correct? Is there an alternative to this method for FanControl to communicate with my Hydro 115i Pro?

**Comment by EvanMulawski** [comment]
I believe you would be able to use SignalRGB or OpenRGB to control the lighting on your other devices, as the H115i Pro (and any other Asetek Hydro AIO model) is not supported by that software. You just wouldn't be able to control the AIO lighting if it's an RGB model.

**Comment by V-Reacher** [comment]
Deleted, because iCUE V5 does have Options to transfer macros easily to the hardware profiles.
